odoo/addons/base/views/ir_qweb_widget_templates.xml

```diff
@@ -23,7 +23,7 @@
         <div itemprop="address" itemscope="itemscope" itemtype="http://schema.org/PostalAddress">
             <div t-if="address and 'address' in fields" class="d-flex align-items-baseline">
                 <i t-if="not options.get('no_marker')" class="fa fa-map-marker fa-fw" role="img" aria-label="Address" title="Address"/>
-                <div class="w-100" itemprop="streetAddress" t-raw="address.replace('\n', options.get('no_tag_br') and ', ' or ('&lt;br/&gt;%s' % ('' if options.get('no_marker') else '')))"/>
```

Comment:
It should not matter it is a span or a div since its parent is a flex container... but this indeed fixes the bug because a portal scss file is styling span instead of div. I am ok with the fix but would not it be more stable to change that scss rule to target span *and* div instead ?

Clearly we should also remove that scss file styling (which should apply to all addresses and not only portal ones if we decide to keep it) but that would be for master. (That commit: https://github.com/odoo/odoo/commit/60d837044a028e578f804570361af128b40a12b6 was a damn bad idea and I should have reviewed it better but well... in the rush of 12.0 release... -_-')

cc @nim-odoo @jasonvanmalder

Reply:
I changed the scss rule as you recommended